src/main/java/core/basesyntax/handler/impl/BalanceTransactionHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.handler.impl;
+
+import core.basesyntax.handler.TransactionHandler;
+import java.util.Map;
+
+public class BalanceTransactionHandler implements TransactionHandler {
```

Comment:
Don't begin class or method implementation with an empty line. There is an empty line after the package declaration that should be removed.